include/irr/core/math/aabb2dSIMD.h

```diff
@@ -88,17 +86,39 @@ class aabb2dSIMDf
 	inline float getArea() const
 	{
 		core::vectorSIMDf a = internalBoxRepresentation + internalBoxRepresentation.zwxx();
-		return a.x* a.y;
+		return a.x * a.y;
 	}
 
 	//! Stores all 4 edges of the box into an array
-	/** \param edges: Pointer to array of 8 edges. */
-	void getEdges(core::vectorSIMDf & edges) const;
+	/**
+	A ----- D
+	|		|
+	|		|
+	B ----- C
+
+	array modified by this function will store edges in this order:
+	AB, BC, AD, DC
+	\param edges: Pointer to array of 4 edges. */
+	void getEdges(core::vectorSIMDf* edges) const
```

Comment:
actually this function is called `getEdges` but its actually `getVertices` if you look at the definition in `aabbox3d<>`
its for getting the corners
(you can rename if you like)